optimum/exporters/openvino/__main__.py

```diff
@@ -367,7 +368,7 @@ class StoreAttr(object):
 
             if tokenizer is not None:
                 try:
-                    export_tokenizer(tokenizer, output)
+                    export_tokenizer(tokenizer, output / OV_TOKENIZER_FOLDER)
```

Comment:
Please, don't do it, it looks very weird `<root>/openvino_tokenizers/openvino_tokenizers.xml` 

Why after exporting from native HF to OpenVINO models / files, the directory layout should be changed? 
We want to keep directory layout as is, but just replace original model files with OpenVINO exported ones.

Reply:
This was already discussed when the openvino tokenizer export was made default and was part of the reason why this modifications was accepted. 

Currently for each tokenizer the following files are being added :
`openvino_detokenizer.bin`
`openvino_detokenizer.xml`
`openvino_tokenizer.bin`
`openvino_tokenizer.xml`

and we think this is cleaner to have it in a directory, as also done for SD models (for example : https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0/tree/main/tokenizer)

Reply:
This was already discussed [here](https://github.com/huggingface/optimum-intel/pull/580#discussion_r1557865119) and [here](https://github.com/huggingface/optimum-intel/pull/580#issuecomment-2063539071)

Reply:
will merge it for now as it's been a while this was supposed to be added, I'm OK to remove it in a following PR but we would need to also remove the tokenizer export being default as well

Reply:
> This was already discussed when the openvino tokenizer export was made default and was part of the reason why this modifications was accepted.
> 
> Currently for each tokenizer the following files are being added : `openvino_detokenizer.bin` `openvino_detokenizer.xml` `openvino_tokenizer.bin` `openvino_tokenizer.xml`
> 
> and we think this is cleaner to have it in a directory, as also done for SD models (for example : https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0/tree/main/tokenizer)

Yes, but the original tokenizer also has 3-5 files for fast and legacy versions in the root:
`tokenizer.json`
`special_tokens_map.json`
`added_tokens.json` (not always)
`tokenizer_config.json`
`tokenizer.model` (for sentencepiece-based tokenizers)

When there is `tokenizer` folder in the original model repo, we already using it for openvino tokenizer as well:
```
sd/
├── feature_extractor
│   └── preprocessor_config.json
├── model_index.json
├── scheduler
│   └── scheduler_config.json
├── text_encoder
│   ├── config.json
│   ├── openvino_model.bin
│   └── openvino_model.xml
├── tokenizer
│   ├── merges.txt
│   ├── openvino_detokenizer.bin
│   ├── openvino_detokenizer.xml
│   ├── openvino_tokenizer.bin
│   ├── openvino_tokenizer.xml
│   ├── special_tokens_map.json
│   ├── tokenizer_config.json
│   └── vocab.json
├── unet
│   ├── config.json
│   ├── openvino_model.bin
│   └── openvino_model.xml
├── vae_decoder

```